ModernWallet/Classes/ViewController/SendReceive/SendToWalletViewController.swift

```diff
@@ -134,6 +138,10 @@ fileprivate extension CashOutToAddressViewModel {
                 .filterNil()
                 .bind(to: assetId),
             
+            isValidAddressAndAmount
+                .asDriver(onErrorJustReturn: false)
```

Comment:
Since isValidAddressAndAmount is of type Driver casting it to driver is redundant.